src/nvim/os/shell.c

```diff
@@ -423,10 +423,11 @@ static bool out_data_decide_throttle(size_t size)
   pulse_msg[1] = (tick > 1) ? '.' : ' ';
   pulse_msg[2] = (tick > 2) ? '.' : ' ';
   if (visit == 1) {
-    msg_putchar('\n');
+    msg_puts("[...]\n");
```

Comment:
@blueyed why did we add brackets here? It looks weird, it should just be `...` I think?

Reply:
@justinmk https://github.com/neovim/neovim/pull/11130#discussion_r330352413

Feel free to remove them.

Reply:
I think seeing `[   ]` though instead of `` (nothing) might help when looking at the history though.